src/components/edit.jsx

```diff
@@ -0,0 +1,254 @@
+import { useSelector } from "react-redux"
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import { Button } from "./Button";
+import MenuItem from '@mui/material/MenuItem';
+import { updateWebsite } from "../api/api";
+import './edit.css'
+
+export const Edit = () => {
```

Comment:
this component is too long. split to to some functions.

Reply:
קיצרתי עכשיו היא באורך של 226 שורות העברתי את הפונקציות של הולידציות לקובץ אחר אין לי איך לקצר עוד...